plugin-dom/translations/dom_cs.desktop

```diff
@@ -0,0 +1,2 @@
+Name[cs]=Panel DOM stromu
```

Comment:
`DOM stom panelu`... but actually I thing all translations for this plugin are not needed, as this is usable just for developers, not for regular users.